script.js

```diff
@@ -1,11 +1,26 @@
 // Create an array called `colors` that contains three colors.
 
+var colors = ['red', 'yellow', 'green'];
 // Log the last item in the array to the console.
+  console.log(colors[2]);
 
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
 
+  for( i = 0; i < colors.length; i++) {
+    print ('My favorite color is' + colors[i]);
+  }
+
 // Create an `instructor` object that contains three key-value pairs.
 
+var instructor = {
+  class: "WDI",
+  subject: "Javasript",
+  day: "Monday"
+};
+
 // Add a `facial-hair` property to `instructor`.
 
+instructor.facialhair= 'long';
+
```

Comment:
super close! use square bracket notation for keys with symbols `instructor['facial-hair']`